core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -642,7 +642,7 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
 
   // For testing.
```

Comment:
minor: I think it isn't just for testing anymore, maybe we should remove this comment since we are updating this function anyways?